util/regexp-assemble/data/932150.data

```diff
@@ -169,10 +172,7 @@ printf
 ptar
 ptardiff
 ptargrep
-python
-python2
-python3
-python3m
+'python(?:[23](?:\.[0-9.]+)?|3m)?
```

Comment:
What is the starting quote trying to achieve here?

Reply:
@fzipi Hmm. Apparently a `'` isn't a special character.

Maybe it was a mistake in the original regexp that wasn't noticed during the original review? Should we assume it's a mistake and remove it? P.S. the original PR author is not responsive on here anymore.

Reply:
@fzipi Removing the `'` broke one of the tests :thinking:

Reply:
Ahh, it might not have been a typo:

> Lines starting with a single quote ' are treated as literals and will not be escaped.

Reply:
Oh, I think I see it now. Maybe it's a regexp-assemble bug?

It turns this:
```
python(?:[23](?: ...
```

into this:

![image](https://user-images.githubusercontent.com/6837454/178320210-e2de0632-2aa7-417e-995e-e8421ae5d391.png)